admin-manual/installation/installation.rst

```diff
@@ -193,8 +193,9 @@ Using 16.04 (Xenial):
 .. code:: bash
 
    sudo wget -O - https://packages.archivematica.org/1.6.x/key.asc | sudo apt-key add -
-   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/1.8.x/ubuntu xenial main" >> /etc/apt/sources.list'
-   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main" >> /etc/apt/sources.list'
+   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/repos/apt/dev-1.8.x-xenial/ ./" >> /etc/apt/sources.list'
+   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/repos/apt/release-0.11-xenial/ ./" >> /etc/apt/sources.list'
```

Comment:
Should we be using `https` instead?

Reply:
Also would it be possible to use `packages.archivematica.org` at all times incl. WIP packages? e.g. `https://packages.archivematica.org/1.8.x-dev` or similar?

Reply:
The jenkins server doesn't have ssl support for now, but that might change in the near future.

Regarding using the same domain for all packages, it can be done. The original intention was to keep them separate from the official releases to avoid breaking any current install without knowledge.